tests/Console/Command/DescribeCommandTest.php

```diff
@@ -144,17 +175,70 @@ public function testExecuteWithoutName()
         ));
     }
 
+    public function testGetAlternativeSuggestion()
+    {
+        $this->setExpectedExceptionRegExp('InvalidArgumentException', '#^Rule "Foo2/bar" not found\. Did you mean "Foo/bar"\?$#');
+        $this->execute('Foo2/bar', false)->getDisplay(true);
+    }
+
+    /**
+     * @param bool   $decorated
+     * @param string $expected
+     *
+     * @dataProvider provideDescribeSetCases
+     */
+    public function testDescribeSet($decorated, $expected)
+    {
+        $this->assertSame(
+            $expected,
+            $this->execute('@testSet', $decorated)->getDisplay(true)
+        );
+    }
+
+    public function provideDescribeSetCases()
+    {
+        return array(
+            array(
+                true,
+<<<EOT
+\033[32mDescription of\033[39m @testSet \033[32mset.\033[39m
+
+ * \033[32mFoo/bar\033[39m \033[37;41mrisky\033[39;49m
+   | Fixes stuff.
+   \033[33m| Configuration: ['functions' => ['foo' => 'bar']]\033[39m
+
+
+EOT
+            ),
+            array(
+                false,
+<<<'EOT'
+Description of @testSet set.
+
+ * Foo/bar risky
+   | Fixes stuff.
+   | Configuration: ['functions' => ['foo' => 'bar']]
+
+
+EOT
+            ),
+        );
+    }
+
     /**
-     * @param bool $decorated
+     * @param string $name
+     * @param bool   $decorated
+     * @param int    $verbosityLevel
      *
      * @return CommandTester
      */
-    private function execute($decorated)
+    private function execute($name, $decorated, $verbosityLevel = OutputInterface::VERBOSITY_NORMAL)
     {
         $fixer = $this->prophesize();
         $fixer->willImplement('PhpCsFixer\Fixer\DefinedFixerInterface');
         $fixer->willImplement('PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface');
 
+        $fixer->configure(array('functions' => array('foo' => 'bar')))->willReturn(null);
```

Comment:
Its not wrong, its a test that it is returned, since we cannot test for the correct pseudo type `void` this is the best we can do.